dotcom-rendering/src/experiments/tests/sign-in-gate-copy-test-variants.ts

```diff
@@ -1,17 +1,17 @@
 import type { ABTest } from '@guardian/ab-core';
 
-export const signInGateCopyTestJan2023: ABTest = {
-	id: 'SignInGateCopyTestJan2023',
+export const signInGateCopyTestRepeatSept2023: ABTest = {
+	id: 'SignInGateCopyTestRepeatSept2023',
 	start: '2023-01-23',
```

Comment:
Looks great. I think this will be a cleaner and easier way to run this. Let's just get the start/end dates matched up with the Frontend PR (I think the data link name there is old still)

```
start: '2023-08-30',
expiry: '2025-12-01',
```